### ISSUE TITLE
Biomes have no localization

### ISSUE THREAD

**Issue by Sunconure11** [Cryptic-Mushroom/Midnight#3]
![screenshot 2018-10-15 15 20 31](https://user-images.githubusercontent.com/5614902/46972998-f8ec4500-d08d-11e8-80f8-cce255ef34d8.png)

**Comment by gegy1000** [comment]
Do biome names get translated with `I18n`?

**Comment by Andromander** [comment]
No. The names of biomes are determined by the String you gave in the Biome Properties, and are not translated, if I recall correctly
